aci/resource_aci_l3extlifp.go

```diff
@@ -61,12 +63,22 @@ func resourceAciLogicalInterfaceProfile() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"tn_netflow_monitor_pol_name": {
+							Type:       schema.TypeString,
+							Optional:   true,
+							Deprecated: "Use tn_netflow_monitor_pol_dn instead",
+						},
+						"tn_netflow_monitor_pol_dn": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
 						},
 						"flt_type": {
 							Type:     schema.TypeString,
 							Required: true,
```

Comment:
is this a required field? because on apic it will set default value of ipv4?

Reply:
It is only required when "relation_l3ext_rs_l_if_p_to_netflow_monitor_pol" is being set by the user. Ideally "tn_netflow_monitor_pol_name" /"tn_netflow_monitor_pol_dn" should be Required but you can't have both.
So I added a check in CustomDIff and also a note in the docs

Reply:
apologies I meant the flt_type attribute

Reply:
Discussed internally